packages/dev-utils/src/devApp/render.tsx

```diff
@@ -193,6 +197,30 @@ export class DevAppBuilder {
     return this;
   }
 
+  /**
+   * Set available languages to be shown in the dev app
+   */
+  setAvailableLanguages(languages: string[]) {
```

Comment:
Was thinking we could infer this, but it's a bit tricky with loose references to translation refs that can contain additional languages. Perhaps it still makes sense to default it to en + any additional languages in added resources?

Reply:
Perhaps it would make sense to move that functionality to `AppManager` instead and remove the option to pass the `availableLanguages` there?

Reply:
Or is the question whether `setAvailableLanguages` could be removed entirely?

Reply:
Yes, it could be populated from resources in AppManager but I would say it's for another PR

Reply:
For an actual app you want to select the specific languages that you want to have available imo, in case you for example solely rely on built-in translations in plugins. Was thinking this could just be a little bit of assistance in the dev app since it doesn't hurt to list extra languages there.

Let's ship as is though and we can revisit later